tests/rt.conf

```diff
@@ -31,6 +31,10 @@ RUN | cpld_control_gfsv17_iau                           | - noaacloud
 RUN | cpld_restart_gfsv17                               | - noaacloud                           |          | cpld_control_gfsv17
 RUN | cpld_mpi_gfsv17                                   | - noaacloud                           |          |
 
+#SFS
+COMPILE | s2swa_32bit_pdlib_sfs  | intel | -DAPP=S2SWA -D32BIT=ON -DHYDRO=ON -DCCPP_SUITES=FV3_GFS_v17_coupled_p8_ugwpv1 -DPDLIB=ON | - noaacloud | fv3 |
+RUN | cpld_sfs_hydro                                    | - noaacloud                           | baseline |
```

Comment:
Since this appears to be the control configuration test for sfs, then I believe a more appropriate test name would be ``cpld_control_sfs``. I don't think that the ``hydro`` is required, if ``hydro=on`` is the default (control) for this configuration.

Reply:
I will remain this name for now since the SFS configuration is not completely decided yet.

Reply:
The name you've chosen does not really align w/ how we name the RT tests. Yes, the configuration may evolve but this would be the "control" for the "sfs w/ hydrostatic=on".